tests/allocate/test_allocate.py

```diff
@@ -553,3 +555,80 @@ def get_model(self):
                     elif (wt1 == 'static_b' and wt2 == 'burst_c') or (wt1 == 'burst_c' and wt2 == 'static_b'):
                         colloc_b += 1
         self.assertLessEqual(colloc_a, colloc_b)
+
+    def test_forecast_threshold_no_usage(self):
+        allocator = ForecastIPCpuAllocator(
+            TestCpuUsagePredictorManager(),
+            ConfigManager(TestPropertyProvider({})),
+            ThresholdFreeThreadProvider(0.1))
+
+        cpu = get_cpu()
+        w0 = get_test_workload(uuid.uuid4(), 8, STATIC)
```

Comment:
the `8` here implicitly rely on the default values of `get_cpu()`. might be better to be explicit by specifying the arguments to `package_count` and `cores_per_package` arguments to `get_cpu`, or by pulling the default value constant